docs/landing-page/master.adoc

```diff
@@ -2,95 +2,178 @@ include::topics/templates/document-attributes.adoc[]
+  background-size:25%
+}
+.swarm {
+  background-image:url(/docs/images/project-select-icons/swarm.svg)
+}
+.vertx {
+  background-image:url(/docs/images/project-select-icons/vertx.svg);
+  background-size:80%
+}
+.node {
+  background-image:url(/docs/images/project-select-icons/nodejs-logo.png);
+  background-size:60%
+}
+#runtimes {
+  flex: 1;
+  background-color: #f5f5f5;
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+#runtimes .row + .row {
+  display: flex;
+}
+#runtimes .runtime {
+  flex-grow: 1;
+  background-color: white;
+  margin: 0 20px;
+  display: flex;
+  flex-direction: column;
+  padding: 40px 20px;
+}
+#runtimes .runtime p {
+  flex-grow: 2;
 }
-</style>
 
-<div _ngcontent-c3="" class="content">
-<section _ngcontent-c3="" id="intro">
-  <div _ngcontent-c3="" class="container">
-    <h1 _ngcontent-c3="">
-      Continuous application delivery,
-    </h1>
-    <h1 _ngcontent-c3="">
-      built and deployed on OpenShift.
-    </h1>
-    <div _ngcontent-c3="">
-      <a _ngcontent-c3="" class="btn btn-lg" href="/docs/getting-started.html">Get Started Now</a>
-    </div>
+.ribbon::before, .ribbon::after {
+  position: absolute;
+  z-index: 0;
+  content: '';
+  display: block;
+  border: 5px solid #8e0000;
+}
+
+.ribbon {
+  width: 120px;
+  height: 120px;
+  overflow: hidden;
+  position: absolute;
+}
+.ribbon span {
+  position: absolute;
+  display: block;
+  width: 220px;
+  padding: 9px 0;
+  background-color: #c00;
+  box-shadow: 0 5px 10px rgba(0, 0, 0, 0.1);
+  color: #fff;
+  font: 700 11px/1 'Overpass', sans-serif;
+  text-shadow: 0 1px 1px rgba(0, 0, 0, 0.2);
+  text-align: center;
+}
+
+.ribbon-top-right {
+  top: -10px;
+  right: -10px;
+}
+.ribbon-top-right span {
+  left: -45px;
+  top: 30px;
+  transform: rotate(45deg);
+}
+.ribbon-top-right::before {
+  border-top-color: transparent;
+  border-right-color: transparent;
+  top: 0;
+  left: 0;
+}
+.ribbon-top-right::after {
+  border-top-color: transparent;
+  border-right-color: transparent;
+  bottom: 0;
+  right: 0;
+}
+</style>
+<div class="jumbotron">
+  <div class="container">
+    <img src="/docs/images/launch_logo.svg" alt="Fabric8 Launcher Logo" class="pull-right">
+    <h2>Fabric8 Launcher documentation</h2>
+    <p class="lead">
+      Read how to get started and setup your environment and all the good stuff that you can do with Fabric8 Launcher.
+    </p>
+    <p>
+      <a class="btn btn-primary btn-lg" href="/docs/getting-started.html" role="button">Getting started</a>
+    </p>
   </div>
-</section>
+</div>
 
-<section _ngcontent-c3="" id="runtimes">
-  <div _ngcontent-c3="" class="container">
-    <div _ngcontent-c3="" class="row">
-      <div _ngcontent-c3="" class="col-md-12">
-        <h1 _ngcontent-c3="">Supported Runtimes</h1>
+<section id="runtimes">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12">
+        <h3>Documentation on the Supported Runtimes</h3>
```

Comment:
Please, replace _on_ with _about_.

Reply:
"on" is correct here, isn't it?

Reply:
The IBM Style Guide, which is the authoritative guide on writing documentation, mandates we do not use it in this context.